### PR TITLE
Doc: Added MAAS UI onboarding documentation

### PR DESCRIPTION
**Pull request by Jay-Topher**
## Done

- Migrated MAAS UI onboarding to `docs`

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Steps for QA.

## Fixes

Fixes: # .

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in main.

## Screenshots

image

### CODE REVIEW

**Comment by ndv99** [comment]
Just so you know, once your PR is ready for review you can add the `Review: Code needed` label if you need a code review, and `Review: QA needed` if you need someone to perform a QA check on your code (if the latter is the case make sure you include some steps for how to check that your changes are working as intended)

File: docs/MAASUI.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# MAAS UI
+
+- Content
```
Comment (ndv99) [inline docs/MAASUI.md:3]: You can probably remove "content" as a bullet point here and just change it to a L2 heading:

```markdown
## Contents
- [Usability](#usability)
- [Code structure](#code-structure)
- etc...
```

File: docs/MAASUI.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      - [ES6](#es6)
+      - [Prettier](#prettier)
+  - [Proxy](#proxy)
+  - [End-to-end](#end-to-end)
+    - [Cypress](#cypress)
+    - [Playwright](#playwright)
+
+## Usability
+
+Our unofficial policy on responsive design in maas-ui is that everything should be clearly visible on all screen sizes, but it doesn't necessarily have to be the most visually appealing on small screens.
```
Comment (ndv99) [inline docs/MAASUI.md:30]:
```suggestion
Our unofficial policy on responsive design in MAAS-UI is that everything should be clearly visible on all screen sizes, but it doesn't necessarily have to be the most visually appealing on small screens.
```

**Review by ndv99 — APPROVED** [review]
Looking good man, congrats on your first PR! :tada: Just a few suggestions below

File: docs/MAASUI.md (hunk 1 above)
Comment (Jay-Topher) [reply docs/MAASUI.md:3]: Sure, thanks.. I'll work on that

File: docs/MAASUI.md (hunk 2 above)
Comment (Jay-Topher) [reply docs/MAASUI.md:30]: Will do!

**Comment by Jay-Topher** [comment]
> Just so you know, once your PR is ready for review you can add the `Review: Code needed` label if you need a code review, and `Review: QA needed` if you need someone to perform a QA check on your code (if the latter is the case make sure you include some steps for how to check that your changes are working as intended)

@ndv99 , for some reason, I was unable to add these labels

**Comment by tmerten** [comment]
> @ndv99 , for some reason, I was unable to add these labels

@Jay-Topher : I'll make sure that you get proper access rights.

**Comment by ndv99** [comment]
@Jay-Topher Just pushed the capitalisation fix, once the checks have finished running you'll be good to merge

**Pull request merged**